.github/workflows/update-chat-snapshots.yml

```diff
@@ -139,12 +392,12 @@ jobs:
         # should not silently overwrite updates to the target branch.
         run: |
           git add packages/react-composites/*.png
-          git commit -m 'Update packages/react-composites ChatComposite browser test snapshots'
-          git pull origin ${{ needs.get_target_branch.outputs.target }} --no-rebase --no-edit
+          git commit -m 'Update packages/react-composites ChatComposite browser test snapshots 4/4'
+          git pull origin ${{ needs.get_target_branch.outputs.target }} --rebase --no-edit
```

Comment:
Rebase is usually not a very good approach for public branches. Do we really need it here?

Reply:
I was running into issues where one task had committed files out of order.